pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/rule/AbstractApexRule.java

```diff
@@ -5,19 +5,13 @@
 package net.sourceforge.pmd.lang.apex.rule;
 
 import net.sourceforge.pmd.RuleContext;
-import net.sourceforge.pmd.lang.LanguageRegistry;
-import net.sourceforge.pmd.lang.apex.ApexLanguageModule;
 import net.sourceforge.pmd.lang.apex.ast.ApexParserVisitor;
 import net.sourceforge.pmd.lang.ast.Node;
 import net.sourceforge.pmd.lang.rule.AbstractRule;
 
 public abstract class AbstractApexRule extends AbstractRule
         implements ApexParserVisitor {
 
-    public AbstractApexRule() {
```

Comment:
(note: these removals are the continuation of https://github.com/pmd/pmd/pull/2617)